config/initializers/exception_notification.rb

```diff
@@ -0,0 +1,14 @@
+require 'exception_notification/rails'
+require 'exception_notification/sidekiq'
+
+EXCEPTION_RECIPIENTS = JSON.parse(Figaro.env.exception_recipients).freeze
+
+ExceptionNotification.configure do |config|
+  config.add_notifier(
+    :email,
+    email_prefix: "[#{APP_NAME} EXCEPTION] ",
```

Comment:
can we add the environment in the prefix?

Reply:
Somewhat out of scope, but this would also be good to do in other places where we use APP_NAME, such as OTPs and user email. It would be good to be able to distinguish environments from the received messages.

Reply:
It doesn't look like you can add the Rails environment or the current host in the subject. We'll need to submit a pull request to the gem. The gem currently only supports adding the controller name and action, and configuring whether or not the exception appears in the subject.

Reply:
Oh wait. We don't need the request to get the host name. We can read from Figaro.

Reply:
Added here: https://github.com/18F/identity-idp/pull/1740/commits/7225dc577711d82d727cd3429b97b003862464df